lib/IRGen/GenDecl.cpp

```diff
@@ -30,7 +30,6 @@
 #include "swift/ClangImporter/ClangModule.h"
 #include "swift/Demangling/ManglingMacros.h"
 #include "swift/IRGen/Linking.h"
-#include "swift/Runtime/HeapObject.h"
```

Comment:
Was this meant to be rebranch only?  Seems like this can go on main as well?

Reply:
I can put it on `main` if you want, I was just going to leave it until the rebranch is merged.